frontend/javascript/components/docupload/file-document.vue

```diff
@@ -56,6 +77,15 @@
           <span class="visually-hidden">{{ i18n.openAttachmentPage }}</span>
         </a>
 
+        <div
+          v-if="highlightRedaction"
+          class="redaction"
+          :class="{ 'redaction__is-redacted': document.is_redacted }">
+          <div class="redaction-badge">
```

Comment:
better to just use a regular bootstrap badge, like `badge text-bg-success` or `badge text-bg-warning`